ztoc/fbs/ztoc09.fbs

```diff
@@ -0,0 +1,50 @@
+namespace v09;
```

Comment:
can we change this to `ztoc09`?

I feel it's hard to figure out that `ztoc` and `v09` are the same kind of folders (except version)

Reply:
Yes, will change.